src/Eppie.CLI/Eppie.CLI/Eppie.CLI.csproj

```diff
@@ -7,14 +7,19 @@
     <Nullable>enable</Nullable>
     <AnalysisLevel>6.0-all</AnalysisLevel>
```

Comment:
Do we need to add
    <EnableNETAnalyzers>true</EnableNETAnalyzers>
    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
here?